src/MonitoringFunctions/DataService/Kusto/IKustoTableRow.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace MonitoringFunctions.DataService.Kusto
+{
+    /// <summary>
+    /// This interface marks model classes that can be inserted into a kusto table as a row.
+    /// </summary>
+    internal interface IKustoTableRow
+    {
```

Comment:
minor: I'm not yet understanding this interface.  Perhaps I need a more thorough review.

Reply:
I use this interface to refer to all the data classes we have. Otherwise, I would have to refer to them as `object` s. This makes the intention of the methods a bit more clear, in my opinion. It implies "this method requires a kusto table row, rather than any object" (even though any object should be good enough to be a kusto table row). 

Let's discuss this if you aren't sure that this is the right way of doing this, so that we can have the best design possible.